docs/kyma-environment-broker/03-03-runtime-operations.md

```diff
@@ -50,9 +50,21 @@ The deprovisioning process contains the following steps:
 | EDP_Deregistration           | Event Data Platform | Done | Removes all entries about SKR from Event Data Platform. | @jasiu001 (Team Gopher) |
 | Remove_Runtime               | Deprovisioning | Done        | Triggers deprovisioning of a Runtime in the Runtime Provisioner. | @polskikiel (Team Gopher) |
 
-## Add provisioning or deprovisioning step
+## Deprovisioning
+
+Each upgrade step is responsible for a separate part of upgrading Runtime dependencies. To properly upgrade the Runtime, you need the data used during the Runtime provisioning. You can fetch this data from the **ProvisioningOperation** struct in the [initialization](https://github.com/kyma-project/control-plane/blob/master/components/kyma-environment-broker/internal/process/kyma_upgrade/initialisation.go) step.
+
+The upgrade process contains the following steps:
+
+| Name                         | Domain         | Status      | Description                                                                            | Owner     |
+|------------------------------|----------------|-------------|----------------------------------------------------------------------------------------|-----------|
+| Upgrade_Kyma_Initialisation  | Upgrade | Done        | Initializes the `UpgradeOperation` instance with data fetched from the `ProvisioningOperation`. | @ksputo (Team Gopher) |
+| Overrides_From_Secrets_And_Config_Step  | Upgrade | Done        | Builds an input configuration for the Provisioner. | @ksputo (Team Gopher) |
+| Upgrade_Runtime              | Upgrade | Done        | Triggers upgrading of a Runtime in the Runtime Provisioner. | @ksputo (Team Gopher) |
 
-You can configure Runtime provisioning and deprovisioning processes by providing additional steps. To add a new provisioning or deprovisioning step, follow these tutorials:
+## Add/provide additional steps
```

Comment:
Please choose only one verb 😄

Reply:
Also, there's something wrong with this doc. There are two Deprovisioning sections now. And still, the metadata section is not updated.